src/hooks/useTxHistory.ts

```diff
@@ -22,6 +22,7 @@ const useTxHistory = (
     () => {
       if (!safeLoaded || !(filter || pageUrl)) return
 
+      // TODO: pass timezone_offset param to the API
```

Comment:
@iamacook passing timezone_offset to CGW should solve the labels being in the wrong place. It probably also localizes the timestamps in the tx list, need to check.

Reply:
Sadly not. The `timezone_offset` is [ignored by the gateway](https://github.com/safe-global/safe-client-gateway/blob/da658a17153d88ce4964bf3889d70ae0bf79ddba/src/routes/transactions/routes.rs#L128) as it's yet to be implemented.